default.nix

```diff
@@ -3,11 +3,21 @@ let
   lib = pkgs.lib;
   es = lib.escapeShellArg;
 
-  standaloneLocker = pkgs.writers.writePython3Bin "standalone-clojure-nix-locker" {
-    libraries = [ pkgs.python3Packages.GitPython ];
-    # We don't care about lines being too long
-    flakeIgnore = [ "E501" ];
-  } ./locker.py;
+  standaloneLocker = let
+    locker = pkgs.writers.writePython3Bin "locker" {
+      libraries = [ pkgs.python3Packages.GitPython ];
+      # We don't care about lines being too long
+      flakeIgnore = [ "E501" ];
+    } ./locker.py;
+  in pkgs.runCommand "standalone-clojure-nix-locker-with-clojure" {
+    buildInputs = [ pkgs.makeWrapper ];
+  } ''
+    makeWrapper ${locker}/bin/locker \
+      $out/bin/standalone-clojure-nix-locker \
+      --prefix PATH : ${lib.makeBinPath [ pkgs.babashka ]}
+  '';
```

Comment:
This feels a bit clumsy - is there a more straightforward way to make babashka available to `locker.py`'s environment?

Reply:
Ah, I don't think there is a more elegant way to do it *here*, because `writePython3Bin` does not have a way to forward e.g. `runtimeInputs` to the lower-level writers it uses.

However, we already pass `runtimeInputs` to `commandLocker` and we could add babashka there. Would that be correct though?

Reply:
Indeed, `locker.py` already relies on `nix-prefetch-git` being on its `PATH`, and that's only added in `commandLocker`.
So at least adding babashka there wouldn't be *more* broken.

Reply:
Ah, I just assumed that `nix-prefetch-git` is always ambiently available for some reason and didn't follow that trace further :sweat_smile: Thanks! Yeah, adding it there is probably fine - pollutes the environment for the given `command` but as you said, we already do that anyway :shrug: Will change accordingly!

Reply:
Pushed! Also fixed a bug that I had introduced with a cleanup I pushed earlier :see_no_evil: 

I guess the "standalone" part of the name is a bit misleading given that it relies on this particular environment where it's used. The name didn't really make sense to me to begin with ... maybe just name it `generateLockfile` or something?

Reply:
tbh with you some of these names make 0 sense to me now. I'm sure we thought of something when we picked them but that is now lost to time and churn. Feel free to change! `generateLockFile` or `lockfileGenerator` are both fine by me.

Reply:
Heh just wanted to go ahead with this and I think I now understand why it's called that, namely because [it is also returned as part of the default expression](https://github.com/bevuta/clojure-nix-locker/blob/master/default.nix#L12). In that context, the name makes more sense. However, if this is supposed to work properly, it should *really* be made self-contained :smile: Just filed https://github.com/bevuta/clojure-nix-locker/issues/8 for this.